_posts/2023-10-23-unidirectional-topic-operator.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: "Unidirectional Topic Operator"
+date: 2023-10-23
+author: federico_valeri
+---
+
+The Topic Operator enhances Strimzi's capabilities by allowing Kafka administrators to manage Apache Kafka topics using Kubernetes custom resources (CRs).
+You can declare the desired topic state, and the Topic Operator will reconcile it with the actual Kafka cluster state.
```

Comment:
Maybe "desire topic configuration" is better? Not sure talking about "topic state" makes sense, or what did you have in mind?